resources/migrations/000_migrations.yaml

```diff
@@ -13133,6 +13188,62 @@ databaseChangeLog:
             constraintName: unique_dimension_field_id
         # this change can roll back automatically.
 
+  - changeSet:
+      id: v46.00-032
+      author: tsmacdonald
+      comment: Added 0.46.0 -- Unique values_card
+      changes:
+        - addUniqueConstraint:
+            tableName: values_card
+            columnNames: parameterized_object_id, parameterized_object_type, parameter_id
+            constraintName: unique_parameterized_object_card_parameter
+
+  - changeSet:
+      id: v46.00-033
+      author: tsmacdonald
+      comment: Added 0.46.0 -- values_card index on connected object
+      changes:
+        - createIndex:
+            tableName: values_card
+            columns:
+              - column:
+                  name: parameterized_object_id
+            indexName: idx_values_card_parameterized_object_id
+
+  - changeSet:
+      id: v46.00-034
+      author: tsmacdonald
+      comment: Added 0.46.0 -- values_card index on connected card
+      changes:
+        - createIndex:
+            tableName: values_card
+            columns:
+              - column:
+                  name: card_id
+            indexName: idx_values_card_card_id
+
+  - changeSet:
```

Comment:
I see lots of examples of FKs being described inline:
```
- column:
    name: card_id
    type: int
    remarks: 'ID of the Card model persisted'
    constraints:
      nullable: false
      unique: true
      referencedTableName: report_card
      referencedColumnNames: id
      foreignKeyName: fk_persisted_info_card_id
      deleteCascade: true
```

Is there a deliberate reason not to do so here?

Reply:
The real answer is that there was no deliberate reason, just my clumsiness with Liquibase.

That said, in the interim John merged [this linter check](https://github.com/metabase/metabase/pull/27234) that prohibits `ON DELETE CASCADE` from being added in an `addColumn` migration, so I think we're forced into doing it this way anyway

Reply:
ah i hadn't seen that. Thanks